packages/generator-rest/src/generator.ts

```diff
@@ -13,71 +13,128 @@ import {
   removeSync,
   writeJsonSync
 } from 'fs-extra';
+import { Project } from 'ts-morph';
```

Comment:
Why do we need ts-morph here?

Reply:
ts-morph? you mean the why using the lib?
Because we need to generate `generated/PriceCalculation/request-builder.ts`

Reply:
Discussed with Frank, if we keep the ts-morph as the way for the odata generator, I would keep the same way for the rest, because it's easy for maintenance.